docs/sftt/sftt-api-spec-new.md

```diff
@@ -957,13 +961,80 @@ If the user is not a super admin.
 
 ### Import Reservations
 
-`POST api/v1/protected/import/reservations`
+!!! missing "This route has not been implemented yet"
 
+`POST api/v1/protected/import/reservations`
 
 ## Leaderboard Router
 
-!!! missing "This router has not been implemented yet"
+The leaderboard router is used to get the information displayed on the leaderboard, which is blocks completed per user. It is public because future releases possibly include a public leaderboard page. Since none of the information here is strictly private the consequences of making it public are minimal.
+
+A completed block is a block for which the last entry is either `complete` or `qa`. The user and team that are referenced in that last entry are the user and team that should be credited with the completion. If the user is `NULL`, no user will be credited with that completion, and the same goes for a `NULL` team field.
+
+### Get Users Leaderboard
```

Comment:
What's the tie breaker for users/teams with the same number of blocks? Even if we're just leaving it up to whatever the database returns we should specify that